lib/scope.js

```diff
@@ -1260,15 +1283,39 @@ module.exports = {
       await scope.page.waitForTimeout( 400 );
     }
 
-    let { tag, type } = field;
     // We use the last match. We wrote a warning for the user if there were
     // multiple matches in the story that the last value will be used. In future,
     // Steps will be able to set this data gradually and we don't want the developer
     // to get confused thinking that an earlier Step was used instead of a later one.
     let row = matching_input_rows[ matching_input_rows.length - 1 ];
     let set_to = row.value;
 
-    // Detect a custom type and get the element of that custom type if it exists
+    await scope.funnel_the_answer( scope, { handle, set_to, field });
+
+    return row;
+  },  // Ends scope.setVariable()
+
+  get_handle_from_selector: async function ( scope, { field }) {
+    /** Get the element's puppeteer handle using the field's selector. */
+
```

Comment:
Nit: a better name for this would be something like `get_handle_from_field`. Selector is too vague to be useful, as anything on the page could be a selector. Saying what the selector will get you is good though.